vue/components/ui/atoms/svg-viewer/svg-viewer.stories.js

```diff
@@ -0,0 +1,17 @@
+import { storiesOf } from "@storybook/vue";
+import { withKnobs, text } from "@storybook/addon-knobs";
+
+storiesOf("Components/Atoms/SVG Viewer", module)
+    .addDecorator(withKnobs)
+    .add("SVG Viewer", () => ({
+        props: {
+            src: {
+                default: text("Src", "")

```

Comment:
```suggestion
                default: text("Source", "")
```
also, please provide an example instead of leaving as empty string. The example may be one that's hosted in our servers